__tests__/github-mention-test.js

```diff
@@ -10,35 +10,81 @@
 jest
   .dontMock('../mention-bot.js')
   .dontMock('download-file-sync')
-  .dontMock('fs');
+  .dontMock('fs')
+  .dontMock('minimatch');
 
 var mentionBot = require('../mention-bot.js');
 var fs = require('fs');
 
 describe('Github Mention', function() {
+
+  function getFile(filename) {
+    return fs.readFileSync(__dirname + '/data/' + filename, 'utf8');
+  }
+
   // If you are working on the algorithm itself, it is useful to be able to run
   // the complete flow that downloads the diff and subsequent blames. Since
   // doing http requests is unreliable in tests, it is disabled by default.
-  xit('CompleteFlow', function() {
-    mentionBot.enableCachingForDebugging = true;
-    var prs = [3238];
-    prs.forEach(function(i) {
```

Comment:
Can you put back this test, it's very useful when you are debugging the algorithm.